lib/components/molecules/ArticleThumbnail/types/index.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+type anchor = {
+  to: string | any,
+  as?: string,
+  children: Node,
+  className?: string,
+  inheritedStyles?: string,
+  handleLinkClick?: (SyntheticEvent<>) => void,
+  shallow?: boolean,
+  scroll?: boolean,
+  target?: '' | '_self' | '_blank' | '_parent' | '_top',
+  forceClick?: boolean,
+};
+
+type image = {
```

Comment:
Don't define the type for image again. Use the one that exist already. 

Props like anchorPorps should not be part of Image.